### PR TITLE
add original date and status to APA style

### PR DESCRIPTION
**Pull request by adam3smith**
@bwiernik @nickbart1980 could you take a quick looks if this looks right to you? Seemed pretty straightforward for APA. If you think this is OK, I'll add this to all APA variants.
I'll do Chicago, which will be a little trickier, next. 
@rmzelle - I'm hoping to add these to a couple of styles: for pandoc users, for people who want to use the Zotero/citeproc workarounds, and for other reference managers that are interested in implementing this. I assume there are no objections?

### CODE REVIEW

File: apa.csl [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
-  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citatinstyles.org/visualEditor/) -->
```
Comment (rmzelle) [inline apa.csl]: Typo?

**Comment by rmzelle** [comment]
Sure.

File: apa.csl (hunk 1 above)
Comment (adam3smith) [inline apa.csl]: yeah, happened while saving. I'll take the line out given that this has now gone many iterations w/o the editor.

**Comment by adam3smith** [comment]
OK, advanced online is added. CSL automatically sorts items with no date last (no matter the sort order), so that's already working correctly automatically.
I'm happy with this now, unless there are further objections, I'm going to accept his later today.

**Comment by bwiernik** [comment]
Looks good to me.

**Comment by adam3smith** [comment]
thanks everyone

**Pull request merged**